<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshPaintUI</class>
 <widget class="QFrame" name="MeshPaintUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="zoneColor">
        <item row="0" column="2">
         <widget class="QCheckBox" name="colorWhite">
          <property name="text">
           <string>White</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="colorBlack">
          <property name="text">
           <string>Black</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="paintColor_rb">
          <property name="text">
           <string>Paint Color</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="boundaryFill">
          <property name="text">
           <string>Boundary Fill</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="fillColor">
          <property name="text">
           <string>Fill Color</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="boundaryDraw">
          <property name="text">
           <string>Draw Boundary</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="zoneChangeSize">
     <item>
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>         Size </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinSizePixel"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
